### PR TITLE
Overview page

### PR DESCRIPTION
**Pull request by JannikUekoetter**
Created dummy data for the page of future objects. Unfortunately, there are currently errors in the display and the images are included at maximum size. Error is suspected at "layout="fill"".

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -12,6 +176,16 @@ export default function Home() {
...
       <Main>
         <h1>My App</h1>
+        <ul>
+          {lostItems.map((lostItem) => (
+            <LostItemPage
+              name={lostItem.name}
+              description={lostItem.description}
+              img={lostItem.img}
+              id={lostItem.id}
+            />
```
[the remark this answers is not in the document]
Comment (JannikUekoetter) [reply pages/index.js:186]: Done

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,171 @@
...
+    },
+    {
+      name: "Golden Jewelry",
+      description:
+        "Found jewelry and intend to keep it. Just posting here for recognition.",
+      img: "/assets/foundItems/thirteen.jpeg",
+      id: 13,
+    },
+  ];
+
```
[the remark this answers is not in the document]
Comment (JannikUekoetter) [reply pages/index.js:168]: Done

**Review by sebsch1337 — APPROVED** [review]
Everything well changed. Move on! :)

File: components/FoundItem.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        This item is: {name}. Its description is: {description}. Its ID is: {id}
+        . It looks like:
+        <ImageContainer>
+          <Image
+            objectFit="cover"
+            src={img}
+            width={200}
+            heigth={200}
+            layout="fill"
+            alt="lost items"
```
Comment (CathrinTruchan) [inline components/FoundItem.js:18]: I would suggest the alt text is "found items" because it's the found items component?

**Review by CathrinTruchan — APPROVED** [review]

**Pull request merged**